scripts/add-at-macro/src/add_at_macro/core.clj

```diff
@@ -0,0 +1,374 @@
+  [loc]
+  (loop [loc (z/down loc)]
+    (cond
+      (z/end? loc) nil
+      (-> loc z/string (= "at")) loc
+      :else
+      (recur (z/right loc)))))
+
+(defn find-require-macros-in-require
+  "Finds `:refer-macros` in a re-com import form which is in the `:require` form, i.e
+  (:require [re-com.core :refer-macros [] ...] ...). If found, we search if the `:refer-macros`
+  option loads `at` macro and we remove it if true. The at-macro will be added to the `:refer`
+  option."
+  [loc]
+  (let [refer-macros  (z/find-value loc z/next ':refer-macros)
+        refer-macros  (z/right refer-macros)
+        at-macro      (when refer-macros
+                        (find-at-loc refer-macros))]
+    (if at-macro
+      (-> at-macro z/remove z/up z/up)
+      loc)))
+
+(defn append-at-to-recom-require-form
+  "Checks if re-com import statement has :refer option, appends 'at' macro if True, and
+  adds refer option with 'at' macro if not. Loc is [re-com.core ...]"
+  [loc verbose?]
+  (let [loc                  (-> loc z/string z/of-string)
+        loc                  (-> loc find-require-macros-in-require)
+        refer-keyword        (z/find-value loc z/next ':refer)
+        refer-namespaces-loc (if-not (nil? refer-keyword)
+                               (-> refer-keyword z/next))
+        referred-namespaces  (if refer-namespaces-loc
+                               (into [] (z/sexpr refer-namespaces-loc)) ;; Referred keywords
+                               [])
+        has-at?              (some #{'at} referred-namespaces) ;; If at macro is already loaded.
+        at+namespaces        (into [] (cons 'at referred-namespaces))
+        recom-alias          (get-alias loc)]
+    {:used-alias          recom-alias
+     :required-namespaces (into [] (remove #{'p 'p-span} referred-namespaces))
+     :current-loc         (if (z/vector? refer-namespaces-loc)
+                            (if-not has-at?
+                              (do
+                                (when verbose?
+                                  (println "Appending at macro."))
+                                (-> refer-namespaces-loc
+                                    (z/replace at+namespaces)
+                                    z/up))
+                              (do
+                                (when verbose?
+                                  (println "Found at macro."))
+                                loc))   ;; recom form already loads at macro
+                            (do
+                              (when verbose?
+                                (println "Adding refer option."))
+                              (-> loc
+                                  (z/append-child (z/sexpr (z/of-string ":refer")))
+                                  (z/append-child (z/sexpr (z/of-string "[at]"))))))}))
+
+(defn remove-at-from-required-macros
+  "Checks if `at` macro is loaded in :require-macros form and removes it if true. "
```

Comment:
Given `loc` is a `:require` vector, will return a modified vector with `at` removed from the `:require-marcos` section , if present. 

For example ... 
  [....blah] 
is transformed into 
  [....blah]
  
  
---

Would suggest tweaking other docstrings into this form, as you see fit.